packages/venia-ui/lib/components/Checkout/overview.js

```diff
@@ -57,11 +56,7 @@ const Overview = props => {
                     onClick={handleAddressFormClick}
                     showEditIcon={hasShippingAddress}
                 >
-                    <ShippingAddressSummary
-                        classes={classes}
-                        hasShippingAddress={hasShippingAddress}
-                        shippingAddress={shippingAddress}
```

Comment:
It may be overkill to have the summaries import this through context. Can probably at least do away with the threading down to the overview and to the editable forms.